src/main/java/eu/eutampieri/catacombs/ui/EndGameState.java

```diff
@@ -0,0 +1,59 @@
+package eu.eutampieri.catacombs.ui;
+
+import eu.eutampieri.catacombs.ui.utils.FontUtils;
+
+import java.awt.*;
+import java.awt.event.KeyEvent;
+
+public class EndGameState extends State {
+
+    private static final float BLINK_DELAY = 1f;
+
+    private final Font font = new Font("Monospace", Font.PLAIN, 40);
+    private float blinkDelayCount;
+    private boolean blink = true;
+    private final DungeonGame game;
+
+    public EndGameState(final DungeonGame game) {
+        super(game);
+        this.game = game;
+    }
+
+    @Override
+    public void update (final int delta) {
+        this.blinkDelayCount *= delta;
```

Comment:
are you sure about *=, should it not be += ?

Reply:
you are right

Reply:
fixed